src/flake8_boolean_trap/plugins.rs

```diff
@@ -21,6 +23,32 @@ fn add_if_boolean(checker: &mut Checker, arg: &Expr, kind: CheckKind) {
     }
 }
 
+/// Returns true if an argument fulfills all whiltelist conditions.
+///
+/// Conditions:
+/// * function name must be explicitly whitelisted
+/// * argument in function call must be one of the first two
```

Comment:
Should it not be _exactly_ the second argument, of two? Why "one of the first two"?

Reply:
booleans can be used as keys, too

Reply:
Oh right!

Reply:
I guess, as an exception, the first argument of `fromkeys` shouldn't be boolean given that it's always a tuple. That's fine though.